includes/class-plugin-manager.php

```diff
@@ -296,6 +296,14 @@ public static function get_managed_plugins() {
 				'PluginURI'   => 'https://github.com/xwp/pwa-wp',
 				'Download'    => 'wporg',
 			],
+			'pym-shortcode'                 => [
+				'Name'        => 'Pym.js Embeds',
+				'Description' => 'A WordPress solution to embed iframes that are responsive horizontally and vertically using the NPR Visuals Team\'s Pym.js.',
+				'Author'      => 'INN Labs',
+				'AuthorURI'   => 'http://labs.inn.org/',
```

Comment:
This should be https rather than http, but it's not a big problem.

Reply:
No worries. Updated in https://github.com/Automattic/newspack-plugin/pull/480 for next week's release. 

[It's worth noting that this info is taken directly from the Pym.js Embeds plugin info, and you have http there.](https://github.com/INN/pym-shortcode/blob/master/pym-shortcode.php#L8) :)

Reply:
Oh weird; I thought we had updated that. Thanks for letting me know!